ga.go

```diff
@@ -88,6 +90,36 @@ func (e *Event) Validate() error {
 	return nil
 }
 
+func callGA(endpoint string, reqBody []byte, meta Meta) (*http.Response, string, error) {
+	c := &http.Client{
+		Timeout: DefaultHTTPClientTimeout,
+	}
+
+	req, err := http.NewRequest(
+		http.MethodPost,
+		fmt.Sprintf(endpoint, meta.APISecret, meta.MeasurementID),
+		bytes.NewReader(reqBody),
+	)
+	if err != nil {
+		return nil, "", fmt.Errorf("error creating new http request: %s", err)
+	}
+	req.Header.Set("User-Agent", UserAgent)
+	req.Header.Set("Content-Type", "application/json")
+
+	res, err := c.Do(req)
+	if err != nil {
+		return nil, "", fmt.Errorf("error calling to Google Analytics: %s", err)
+	}
+
+	var resBody []byte
+	resBody, err = ioutil.ReadAll(res.Body)
```

Comment:
I usually get a deprecation warning in my IDE that `ioutil.ReadAll` has been replaced by `io.ReadAll`. Do you want to make that change?